components/NewOutivityForm/index.js

```diff
@@ -18,14 +29,31 @@ export default function NewOutivityForm({ createOutivity, handleCancel }) {
           <StyledNewOutivitiesFormField>
             <label htmlFor="outivityImage">Image</label>
 
+            {selectedImage && (
+              <>
+                <StyledFormPreviewImage
+                  alt="imagePreview"
+                  name="imagePreview"
+                  width={300}
+                  height={200}
+                  src={selectedImage}
+                />
+                <StyledFormRemoveButton
+                  type="button"
+                  onClick={() => setSelectedImage(null)}
+                >
+                  Remove
+                </StyledFormRemoveButton>
```

Comment:
A minor point, but at the moment this will only remove the preview image. It won't remove the contents of the input for the file.

Reply:
Thanks for the hint @onemanwenttomow !
We tried several options but in the end only the preview image was removed..

Do you have any idea what we're overlooking?

Reply:
Thank you @vlatkojuric !! 

In the meantime we've also managed to make some progress here. So now both the image and the input are removed :)